src/main/scala/rocket/Tile.scala

```diff
@@ -129,11 +131,14 @@ class RocketTile(tileId: Int)(implicit p: Parameters) extends LazyModule {
 
       ptwPorts ++= roccs.flatMap(_.io.ptw)
       uncachedArbPorts ++= roccs.flatMap(_.io.utl) // TODO no difference between io.autl and io.utl for now
-    }
 
-    val uncachedArb = Module(new ClientUncachedTileLinkIOArbiter(uncachedArbPorts.size)(icacheParams))
-    uncachedArb.io.in <> uncachedArbPorts
-    ucLegacy.module.io.legacy <> uncachedArb.io.out
+      val uncachedArb = Module(new ClientUncachedTileLinkIOArbiter(uncachedArbPorts.size)(icacheParams))
+      uncachedArb.io.in <> uncachedArbPorts
+      ucLegacy.module.io.legacy <> uncachedArb.io.out
+    } else {
+      ucLegacy.module.io.legacy.acquire.valid := Bool(false)
+      ucLegacy.module.io.legacy.grant.ready := Bool(false)
+    }
```

Comment:
Nope, RoCC is now the only thing. I was going to make a separate PR that provides a LegacyRoCC diplomatic wrapper around old RoCC instances. 

Going forward, I agree it's time revisit what we actually want the RoCC interfaces to look like.  I think at the moment things are in particular disarray because of the non-upstreaming of the custom instruction stuff (e.g. #467?) so it might be a good time to re-spec out that side of the interface as well as the memory side? We probably should also cake-ify the Tile to make it easier to add arbitrary numbers and mixes of RoCCs.